docs/source/commands/license.mdx

```diff
@@ -35,5 +35,5 @@ rover license fetch --graph-id <apollo-graph-id>
 
 #### Related topics
 
-* The (online) [Enterprise license](/router/enterprise-features/#offline-enterprise-license) that a router connected to GraphOS gets from [Apollo Uplink](/federation/managed-federation/uplink/)
-* [Offline Enterprise license](/router/enterprise-features/#offline-enterprise-license)
+* The (online) [Enterprise license](/graphos/routing/license#offline-license) that a router connected to GraphOS gets from [Apollo Uplink](/federation/managed-federation/uplink/)
```

Comment:
This links to a page with header "offline license" but it says `(online)` in the sentence, which is confusing...

Reply:
Good catch!

```suggestion
* The (online) [Enterprise license](/graphos/routing/graphos-features#the-enterprise-license) that a router connected to GraphOS gets from [Apollo Uplink](/federation/managed-federation/uplink/)
```